libs/api/ebpf_api.cpp

```diff
@@ -174,9 +183,10 @@ _Requires_lock_not_held_(_ebpf_state_mutex) inline static ebpf_map_t* _get_ebpf_
 
     EBPF_RETURN_POINTER(ebpf_map_t*, map);
 }
+CATCH_NO_MEMORY(NULL)
```

Comment:
This will result in a mismatched log entry where line 173 logs entry and there is no corresponding log exit.
Instead the CATCH macro needs to log exit.